snap/snapcraft.yaml

```diff
@@ -16,17 +16,17 @@ system-usernames:
 
 layout:
   /etc/mongod.conf:
-    symlink: $SNAP_COMMON/mongod.conf
+    symlink: $SNAP_DATA/etc/mongod/mongod.conf
```

Comment:
Why is that line needed? I am not sure

Reply:
This likely isnt needed.  I am fairly certain that the layout here is not utilized and can be removed.  This would need to be tested though.

Reply:
Good to know, testing now :)